pkg/testutil/testca/testca.go

```diff
@@ -65,6 +65,7 @@ func New(t testing.TB) *CA {
 		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth},
 		KeyUsage:              x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 		BasicConstraintsValid: true,
+		IPAddresses:           []net.IP{net.ParseIP("127.0.0.1")},
```

Comment:
Shouldn’t this be passed to ca.NewCert() ?

Reply:
Thanks